DbConnection/src/main/java/com/solvd/laba/db/model/Rental.java

```diff
@@ -0,0 +1,127 @@
+package com.solvd.laba.db.model;
+
+import java.util.Date;
+
+public class Rental {
+    private int id;
+    private Date rentalDate;
+    private Date returnDate;
+    private int customerId;
+    private int insuranceId;
+    private int rentalRateId;
+    private int vehicleId;
+    private int promotionId;
+    private int employeeId;
+
+    public Rental(int id, Date rentalDate, Date returnDate, int customerId, int insuranceId,
```

Comment:
Yeah I created an extra custom constructor without an id as a parameter because my database primary key is set to auto increment. I was able to update the table with that constructor. I just realized after your comment, that it is unnecessary so I removed all those extra constructors.